src/main/java/com/coherentsolutions/training/aqa/java/web/nikonova/objects/AuthenticationPage.java

```diff
@@ -0,0 +1,65 @@
+package com.coherentsolutions.training.aqa.java.web.nikonova.objects;
+
+import com.coherentsolutions.training.aqa.java.web.nikonova.browsers.BrowsersSettings;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebDriverException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+
+public class AuthenticationPage extends BasePage{
+
+    @FindBy(css =  "#create-account_form")
+    private WebElement accountCreationForm;
+
+    @FindBy(css = "#email_create")
+    private WebElement registrationEmailInput;
+
+    @FindBy(css = "#SubmitCreate")
+    private WebElement registerButton;
+
+    @FindBy(css = "#passwd")
+    private WebElement passwordInput;
+
+    @FindBy(css = "#email")
+    private WebElement emailInput;
+
+    @FindBy(css = "#SubmitLogin")
+    private WebElement signinButton;
+
+
+    public AuthenticationPage(WebDriver driver) {
+
+        super(driver);
+    }
+
+    public UserRegistrationFormPage registerAccount(String email) {
+
+        this.registrationEmailInput.sendKeys(email);
+        this.registerButton.click();
```

Comment:
The 'this' keyword is primarily used in three situations. The first and most common is in setter methods to disambiguate variable references. The second is when there is a need to pass the current class instance as an argument to a method of another object. The third is as a way to call alternate constructors from within a constructor.